docs/source-pytorch/common/trainer.rst

```diff
@@ -1064,15 +1075,15 @@ With :func:`torch.inference_mode` disabled, you can enable the grad of your mode
 
 .. code-block:: python
 
-    class LitModel(LightningModule):
+    class LightningTransformer(pl.LightningModule):
         def validation_step(self, batch, batch_idx):
-            preds = self.layer1(batch)
+            preds = self.model(input.view(1, -1), target.view(1, -1))
```

Comment:
It is unclear to me why we reshape the input like this.

Reply:
updated in [05459ca](https://github.com/Lightning-AI/lightning/pull/18452/commits/05459caf327ed834c88f61e1dbefcc4a0a4b240f)